assets/src/web-stories-block/css/style.css

```diff
@@ -0,0 +1,9 @@
+@import 'variables.css';
+@import 'common.css';
+@import 'lightbox.css';
+
+/* View Types */
+@import 'views/grid.css';
+@import 'views/circles.css';
+@import 'views/list.css';
+@import 'views/carousel.css';
```

Comment:
Will webpack actually import these, so that all the CSS is eventually in this file in the end?

If so, great!

Reply:
yes, webpack will build all this into a single css file.

Reply:
As far as I understand, that is true, it builds them all in one file.

Reply:
Also confirmed by building locally.